### ISSUE TITLE
Expand the Factory and EasyScore APIs

### ISSUE THREAD

**Issue by LukeDowell** [vexflow/vexflow#247]
I feel like there is a lot of room to add some ergonomic improvements to the `Flow.Factory` and EasyScore related areas of vexflow. I'd like to add / have added:

* The ability to easily chain staves (read, measures) together.
* The ability to have chained staves wrap if the drawing area is too small. I'd basically like to be able to give the System a list of treble measures and cleff measures, and have it give me a reasonably formatted staff by default.
* Denoting measures in the easyscore format, something like "C4/h, E4, |, G4, C5"

I think it would be powerful to be able to provide a simple string of voicings to vexflow, and to receive a properly formatted sheet of music by default. 

I'd expect all of these to be opt-in rather than changing any kind of existing default behavior. As it stands, as someone who has tinkered with vexflow, I think the higher level API is extremely limited and I'd like to make that better.

I'd like to be able to do something like:
```
system.setStaveAlignment(StaveAlignment.MEASURES)
system.wrapStaves(true)
``` 

and then whenever system.addStave is called, these two configurations modify the x and y positions of each stave in the system so that they are nice and aligned.

Please let me know if I am missing something or if I'm totally off base. Thanks!

**Comment by LukeDowell** [comment]
I did just notice this discussion which seems related: https://github.com/vexflow/vexflow/issues/158

**Comment by rvilarl** [comment]
Yes, #158 is related. We are promoting Factory as the main API and provide EasyScore as a helper. It would be great to get your help on these features.
